### ISSUE TITLE
Can't export any webm file

### ISSUE THREAD

**Issue by eCinS** [fnordware/AdobeWebM#30]
So I was looking for a software that can enceode webms and stumbeled on your plugin. I have a lot of webm files and I want to edit them and then export them back in the same webm format. Adobe Premiere CS6 opens them without problem and I can edit and do whatever I want with them, but when I try to export/render them I get the unknown error - can't compile movie. It doesn't matter in which format I try to export the same error. At first I thought that maybe the problem was with webms I was using, but it can't be that they're all broken?

**Comment by fnordware** [comment]
Strange. You say you're using Premiere CS6. Which OS?

**Comment by eCinS** [comment]
Well I guess the problem would be windows 10. I had nothing, but problems with it. Sound stopping, videos lagging, wallpaper breaking. So I wouldn't wonder if it also affects your plugin.

**Comment by Baccanno** [comment]
Same issue exporting any format from webm files, using CC 2015 on win8.1

**Comment by Baccanno** [comment]
Seems to start exporting things ( I have several seconds of a webm file exported ) but then crashes with the error

**Comment by fnordware** [comment]
I wonder if this is a bug Adobe has recently introduced. Try exporting [this WebM file](http://media.xiph.org/mango/tears_of_steel_1080p.webm) and [this MP4 file](http://www.fnordware.com/OpenEXR/DWAB_test.mp4).

Someone sent me an MP4 that causes Premiere to crash immediately. It's Adobe's bug, but maybe I should publish a version that works around it.

If those files work but yours doesn't, maybe you can post it somewhere or get it to me some other way?

**Comment by luke80** [comment]
I'm experiencing a similar issue with my webm source file converting to any other format.
````
 - Encoding Time: 00:00:33
02/18/2016 02:49:20 PM : Encoding Failed
-------------------------------------------------------
Export Error
Error compiling movie.

 Unknown error.

-------------------------------------------------------
 - Encoding Time: 00:00:31
02/18/2016 02:50:22 PM : Encoding Failed
-------------------------------------------------------
Export Error
Error compiling movie.

 Unknown error.

-------------------------------------------------------
 - Encoding Time: 00:00:29
02/18/2016 03:02:20 PM : Encoding Failed
-------------------------------------------------------
Export Error
Error compiling movie.

 Unknown error.

```

I don't have the problem when converting your tears_of_steel_108p.webm file. Mine comes from join.me screen recording feature. It plays perfectly well in a browser, but lacks the audio track. I'm trying to use Adobe to mix together the video and audio, and join.me only provides webm by default. I'd prefer not to re-record with something more functional and was hoping your codecs would save me.

Using:
Adobe Media Encoder CC 7.2.2.29 (64-bit)

**Comment by luke80** [comment]
Also of note - Premiere can play the video files in the workspace, but fails upon export.

Premiere Pro CC 7.2.2 (33)

**Comment by luke80** [comment]
Windows 8.1 Enterprise

**Comment by fnordware** [comment]
I'm going to need to be able to replicate it on my end—I'll need some footage and maybe a project.

You could also file your bug with Adobe. They do take a look at bugs that are plug-in related.

http://www.adobe.com/products/wishform.html

**Comment by cnfairfax** [comment]
Wanted to comment to say that I'd be happy to provide you with footage and project info if you still need it. I had the same issue using the most recent version of CC. Using Premiere Pro 2014 for now, but I'd love to see this plugin working on the most recent updates.

**Comment by fnordware** [comment]
Yes, please! This bug is still open on Adobe's end because they haven't been able to reproduce it. Ideally you'd pass along footage, a project (if necessary - not necessary if you get the crash from Media Encoder or exporting your source without going through a timeline), and as much information about your computer setup as possible.

**Comment by TAMHAN** [comment]
Hello,
I just wanted to confirm that the error also occurs here!

All the best
Tam

**Comment by TAMHAN** [comment]
Premiere CC 10.3.0 on Windows 8.1 Enterprise. The WebM file in question was produced by VirtualBox

**Comment by ammcintosh1** [comment]
I'm getting the same error, using Premier CC 9.0.1 on OS X 10.10.5

**Comment by Scottapotamas** [comment]
Also getting the same error on export. I've been able to view/edit the files in the timeline but get crashes on export from Pr or media encoder. 

I also have Pr lock up for 20-30 seconds when trying to import, preview in the bin or add any webm to the timeline. When this occurs Premiere transitions into "application not responding" state, and RAM usage climbs from ~14GB to ~22GB over a 10-15 second period. Once the clip loads the program runs normally and the allocated ram appears to be released.

My webm came from the free version of http://icecreamapps.com/Screen-Recorder/ on Windows10 1607.

**Comment by rag000** [comment]
Possibly related: Was getting similar errors trying to transcode to WebM using MediaEncoder CC (2017) Mac OS 10.11.6

My goal was to convert an MXF to MP4 and WebM with the audio track set to mono. Converting to MP4 was successful. Converting to WebM was not. Tried converting the MP4 (mono) with no luck. What did seem to eventually work (in terms of avoiding the error) was downgrading to WebM_Premiere_v1.0.2 from v1.0.3 and rendering out the mono MP4 as stereo WebM.

**Comment by fnordware** [comment]
Hey guys, I'd like to look into your problems, but I'll need to know your settings if I can hope to replicate them. Oftentimes I'll need a source file too.

**Comment by TAMHAN** [comment]
Hello,
Shoot me an email off the list and I will send you a sample project!

Tam
-- 
With best regards
Tam Hanna
---

NEW: Enjoy electronics? Like seeing oscilloscopes get repaired? Please 
subscribe to my new YouTube channel -> http://www.youtube.com/user/MrTamhan

**Comment by SonofAriston** [comment]
Hi fnordware, I'm having similar problems and can give you any of my settings and source or project files. I'm running Premiere Pro CC (2017) on my 10.11.6 MacBook Pro. After several rounds of trial and error, I'm confident the exporting bug is reproducible whenever exporting a sequence that contains certain webm files. I tried three completely different webm files and had the same problem. Strangely, however, your tears_of_steel file DID work!? Exporting mp4 sources to mp4 works fine, but if there are webm clips in the sequence (apparently other than tears) the export fail, whether you're trying to export as webm or mp4, and seemly regardless of how you configure your sequence settings. Possibly related: Premier seems to have a really hard time rendering even tiny webm clips in the program monitor. I can scrub a 4k mp4 without issue, but even simply trying to play through a webm causes lags and freezing. I'm wondering if the webm files I have are corrupted.. I downloaded them from youtube with youtube-dl, and Imported them to premier which passed them through Media Encoder CC (2017) before adding them to my project.

**Comment by phfatmonkey** [comment]
Hi, same problem. I can provide a project and video file, it's too big to post here though (26 MB). 

I tried converting a WEBM file to any format and it fails after encoding a few frames.

Happens in Adobe Media Encoder, After Effects and Premiere.

- Using Adobe CC 2017
- Windows 10 Home
- 32GB RAM
- NVIDIA GeForce GTX 980
- Captured WEBM video with Screencastify Lite-- Chrome Extension

The After Effects project will sometimes work. I'm not sure why or how.

**Comment by Strangelens** [comment]
The WebM format through premiere on Windows 10 Home and Adobe CC 2017 has extraordinary encoding times that leads me to believe it's not working properly. I have basically the same specs as the above comment. Weirdly enough, I loaded it to my mac, and voila, it was running much faster.
